enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
Also what I was thinking above is that we *could* add a `contact_enterprise_customer_user_id` FK field to the `EnterpriseCustomer` record.  This FK would point to an `EnterpriseCustomerUser` record, and we could determine the actual contact info based on the subsequent relationship-hop to the actual `User` and `UserProfile` records.  Hopefully that's not too crazy...